submissions/api.py

```diff
@@ -48,64 +52,122 @@
 TOP_SUBMISSIONS_CACHE_TIMEOUT = 300
 
 
-def create_submission(student_item_dict, answer, submitted_at=None, attempt_number=None, team_submission=None):
-    """Creates a submission for assessment.
+def create_external_grader_detail(submission, event_data):
```

Comment:
Nit: You can probably get away with passing in just the `submission_id` so that the callers can use a Submission object if they have it, but could also just pass in `submission_id` if all they have is a foreign key. (You can do `objects.create(submission_id=submission_id ... ` and it will do the right thing.